babel.config.js

```diff
@@ -33,6 +33,12 @@ const defaultAlias = {
   '@material-ui/lab': './packages/material-ui-lab/src',
   '@material-ui/styles': './packages/material-ui-styles/src',
   '@material-ui/utils': './packages/material-ui-utils/src',
+  /**
+   * required to support forwardRef
+   * the breaking change in hnrs was the dropped support for react 0.13 which we
+   * don't do anyway so we just force our version of hnrs on other packages.
+   */
+  'hoist-non-react-statics': './packages/material-ui/node_modules/hoist-non-react-statics',
```

Comment:
It's not pretty. But we can only choose between this, convincing every dependency to drop react<0.14 i.e. moving to `hnrs@3.x` or convincing `hoist-non-react-statics` to backport v3 to v2. See mridgway/hoist-non-react-statics#65.